cli/command/image/build.go

```diff
@@ -146,7 +146,16 @@ func NewBuildCommand(dockerCli command.Cli) *cobra.Command {
 	flags.StringVar(&options.imageIDFile, "iidfile", "", "Write the image ID to the file")
 
 	command.AddTrustVerificationFlags(flags, &options.untrusted, dockerCli.ContentTrustEnabled())
-	command.AddPlatformFlag(flags, &options.platform)
+
+	flags.StringVar(&options.platform, "platform", os.Getenv("DOCKER_DEFAULT_PLATFORM"), "Set platform if server is multi-platform capable")
```

Comment:
Can you document this environment-variable? https://github.com/docker/cli/blob/master/docs/reference/commandline/cli.md#environment-variables

Reply:
Added. Note that this PR does not add this env, but just moves around the code.